🔄 Acessando o sistema...
✅ Acessando o sistema realizado com sucesso.
🖼️ Screenshot: screenshots\acessando_o_sistema_153129.png
📸 Sistema acessado.
🖼️ Screenshot: screenshots\url_acessada_153130.png
🔄 Realizando login...
✅ Realizando login realizado com sucesso.
🖼️ Screenshot: screenshots\realizando_login_153133.png
📸 Login realizado.
🖼️ Screenshot: screenshots\login_concluido_153138.png
🔄 Ajustando zoom...
✅ Ajustando zoom realizado com sucesso.
🖼️ Screenshot: screenshots\ajustando_zoom_153138.png
🔄 Abrindo menu Conciliação Bancária...
✅ Abrindo menu Conciliação Bancária realizado com sucesso.
🖼️ Screenshot: screenshots\abrindo_menu_conciliação_bancária_153142.png
📸 Menu 'Conciliação Bancária' aberto.
🖼️ Screenshot: screenshots\menu_aberto_153142.png
🔄 Acessando formulário de cadastro...
✅ Acessando formulário de cadastro realizado com sucesso.
🖼️ Screenshot: screenshots\acessando_formulário_de_cadastro_153145.png
📸 Formulário de cadastro aberto.
🖼️ Screenshot: screenshots\formulario_aberto_153145.png
🔄 Clicando no botão Salvar...
✅ Clicando no botão Salvar realizado com sucesso.
🖼️ Screenshot: screenshots\clicando_no_botão_salvar_153147.png
📸 Clique no botão Salvar realizado.
🖼️ Screenshot: screenshots\apos_salvar_153147.png
Traceback (most recent call last):
  File "C:\Users\Gold System\AppData\Local\Temp\_MEI282802\cenariostestespegasus\CadastrosPrincipais\CadastrosCenáriosConciliaçãoBancária\cadastrodeconciliaçaobancaria3ºcenario.py", line 178, in <module>
    main()
    ~~~~^^
  File "C:\Users\Gold System\AppData\Local\Temp\_MEI282802\cenariostestespegasus\CadastrosPrincipais\CadastrosCenáriosConciliaçãoBancária\cadastrodeconciliaçaobancaria3ºcenario.py", line 157, in main
    _, tipo_alerta = encontrar_mensagem_alerta(driver, doc)
                     ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
TypeError: encontrar_mensagem_alerta() takes 0 positional arguments but 2 were given
